### ISSUE TITLE
Contacts from filtered out phonebooks are displayed in the Favorites.

### ISSUE THREAD

**Issue by KarlGeorg** [SimpleMobileTools/Simple-Contacts#919]
**Environment:**
- Android 8.1.0
- Funtouch OS (Vivo phone 1718)
- Simple Contact v6.22.0 pro

**Steps to reproduce:**
1. I select only my main-phonebook in the phonebook-filter (Menu -> Filter) and deselect e.g. the Signal-messanger-phonebook. 
2. Then I set a contact as favorite (via the star).

**Actual result:**
- Favorites now shows 2 entries from this contact: one from the main phonebook and one from the Signal phonebook.

**Expected result:**
- The main phonebook entry is displayed.
- The Signal entry is not displayed.

**Question ;) Is this a bug?**

**main phonebook entry**
![contact phone book](https://user-images.githubusercontent.com/8435583/210741611-6087cc51-af3c-451c-8ceb-eab8a4e1056b.jpg)

**signal phonebook entry**
![contact signal](https://user-images.githubusercontent.com/8435583/210741615-e9cbdbe3-a5b4-4690-ac4c-859a72977312.jpg)

**favorites tab**
![favorites](https://user-images.githubusercontent.com/8435583/210741616-bb05f227-7472-403c-aac8-46e6a0390f56.jpg)

**phonebook filter**
![phone book filter](https://user-images.githubusercontent.com/8435583/210741618-7b040d9e-6088-4f65-b90f-18a8bc353314.jpg)

**Comment by Aga-C** [comment]
I've one contact with 2 contact sources (it's a one entry):
![image](https://user-images.githubusercontent.com/85929121/210749297-7e928162-ced6-4649-be78-0f1582bd97f5.png)

I think `Manage duplicate contacts` option has something to do with it.

**Comment by KarlGeorg** [comment]
> I've one contact with 2 contact sources (it's a one entry): ![image](https://user-images.githubusercontent.com/85929121/210749297-7e928162-ced6-4649-be78-0f1582bd97f5.png)
> 
> I think `Manage duplicate contacts` option has something to do with it.

True, I don't merge the contact sources - so it's more of a workaround. The point is that this way (merged) I can't find out from which source possible wrong/different entries come from. -> Could this be a new feature to list all phonebooks for merged contacts?
